<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>windowChatForm</class>
 <widget class="QWidget" name="windowChatForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>966</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="currentLoginLable">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>141</width>
         <height>21</height>
        </size>
       </property>
       <property name="text">
        <string>Текст сообщения: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="messagesLineEdit">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>29</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="senMessagesButton">
       <property name="minimumSize">
        <size>
         <width>181</width>
         <height>29</height>
        </size>
       </property>
       <property name="text">
        <string>Отправить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeClientButton">
       <property name="minimumSize">
        <size>
         <width>171</width>
         <height>29</height>
        </size>
       </property>
       <property name="text">
        <string>Выйти из клиента</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextBrowser" name="privateChatBrowser">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>messagesLineEdit</tabstop>
  <tabstop>senMessagesButton</tabstop>
  <tabstop>closeClientButton</tabstop>
  <tabstop>privateChatBrowser</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
